tools/files.js

```diff
@@ -224,13 +229,13 @@ files.getSettings = function (filename, watchSet) {
 // user. Presently, the main thing it does is replace $HOME with ~.
 files.prettyPath = function (p) {
   p = files.realpath(p);
-  var home = process.env.HOME;
+  var home = files.getHomeDir();
   if (! home)
     return p;
-  var relativeToHome = path.relative(home, p);
-  if (relativeToHome.substr(0, 3) === ('..' + path.sep))
+  var relativeToHome = files.pathRelative(home, p);
+  if (relativeToHome.substr(0, 3) === ('..' + files.pathSep))
```

Comment:
we have like a string comparison business on paths going on here. It doesn't necessarily work on case-insensitive filesystems (not a big deal though, prettyPath is for display purposes only)